tool/tctl/common/status_command.go

```diff
@@ -101,6 +101,11 @@ func (c *StatusCommand) Status(client auth.ClientI) error {
 			}
 			info := fmt.Sprintf("%v CA ", strings.Title(string(ca.GetType())))
 			rotation := ca.GetRotation()
+			if rotation.Phase == types.RotationPhaseStandby && len(ca.GetAdditionalTrustedKeys().SSH) > 0 {
```

Comment:
Nit: Let's add a comment please explaining why this condition is the way it is (cause there should be no "additional keys" in standby mode other than in scenario when it's a new hsm auth server).

Reply:
Added a comment here